services/core-web/src/components/common/wrappers/ScrollContentWrapper.js

```diff
@@ -10,30 +10,40 @@ import { PropTypes } from "prop-types";
 const propTypes = {
   id: PropTypes.string.isRequired,
   children: PropTypes.element.isRequired,
-  location: PropTypes.shape({ hash: PropTypes.string }).isRequired,
   title: PropTypes.string.isRequired,
+  history: PropTypes.shape({
+    location: PropTypes.shape({ state: PropTypes.shape({ activeRoute: PropTypes.string }) }),
+  }).isRequired,
 };
 
-export const ScrollContentWrapper = (props) => {
-  const isActive = () =>
-    includes(props.location.hash, props.id) ? "circle purple" : "circle grey";
+class ScrollContentWrapper extends Component {
+  isActive = () => {
+    const activeRoute =
+      this.props.history && this.props.history.location && this.props.history.location.state
+        ? this.props.history.location.state.activeRoute
+        : undefined;
+    const isActiveRoute = includes(activeRoute, this.props.id);
```

Comment:
In this spot, if the 'this.props.history.location.state.activeRoute' doesn't exist( in the case where the user bookmarks the tab, and the '/#state-of-land' exist in the URL) You should be able to check the URL for that search term and use it as the second option if it exists instead of using `undefined`

Reply:
Good idea. I tried to do that earlier (on page load and if URL fragment present, go to corresponding NoW section), but I noticed an issue (see comment above) where URL fragments are being removed. It would be good to set that up!